app/config/parameters.yml.dist

```diff
@@ -5,23 +5,12 @@
 parameters:
     # this demo application uses an embedded SQLite database to simplify setup.
     # in a real Symfony application you probably will use a MySQL or PostgreSQL database
-    database_driver:   pdo_sqlite
-    database_host:     127.0.0.1
-    database_port:     ~
-    database_name:     ~
-    database_user:     root
-    database_password: ~
-    # the 'database_path' is only used for SQLite type databases
-    database_path:     '%kernel.root_dir%/data/blog.sqlite'
+    # the path must be relative or else it will not work on Windows
+    database_url:      'sqlite:///%kernel.root_dir%/data/blog.sqlite'
 
     # Uncomment these lines to use a MySQL database instead of SQLite:
     #
-    # database_driver: pdo_mysql
-    # database_host: 127.0.0.1
-    # database_port: ~
-    # database_name: symfony_demo
-    # database_user: root
-    # database_password: ~
+    # database_url: mysql://root@pass127.0.0.1:3306/symfony_demo
```

Comment:
@dzuelke BTW, seems you missed a colon between password and host (`mysql://root@pass:127.0.0.1:3306/symfony_demo`), didn't you?